### PR TITLE
Fixes sejtalk

### PR DESCRIPTION
**Pull request by sejtalk**
Changelog:
- Using Touch of Elune on other characters boosts their opinion of you.
- You no longer can get both friendly and unfriendly spirits modifier.
- Joining Sisterhood of Elune fires correct event now.
- Yogg-Saron no longer hates eunuchs.
- Event WCCC.70 now has correct localization.
- You can no longer party with Ancients.
- You can no longer get offmap power without Jade Dragon.

### CODE REVIEW

**Comment by MemeWizard** [comment]
>Like if someone else could make this modifier appear.

Yeah, "tidesage blessing" modifiers appear even if i don't press decision.

**Comment by sejtalk** [comment]
> Is it ok that `request_potion` requires 50 gold, but costs 0?
> `summon_ancient` requires 600 piety, but costs 300?

Fixed the `request_potion` and `summon_ancient` is WAD.

**Comment by sejtalk** [comment]
Landless characters can no longer use 'honor_local_spirits' and 'tidesages_blessing_decision' and they now check for province modifier.

File: common/objectives/00_focuses.txt [hunk 1]
```diff
@@ -202,7 +202,7 @@ focus_seduction = {
 			trait = creature_sayaadi
 		}
 		is_uncursed_titanforged_trigger = no
-
+		is_plant_race = no
```
Comment (zumbak04) [inline common/objectives/00_focuses.txt]: is_plant_race trigger doesn't exists.
```suggestion
		plant_race = no
```

File: common/objectives/00_focuses.txt [hunk 2]
```diff
@@ -500,6 +500,7 @@ focus_hunting = {
 		can_rule_peacefully_trigger = yes
 		evil_public_religion_trigger = no
 		NOT = { religion_group = druidism_group }
+		is_plant_race = no
```
Comment (zumbak04) [inline common/objectives/00_focuses.txt]: is_plant_race trigger doesn't exists.
```suggestion
		plant_race = no
```

File: common/objectives/00_focuses.txt [hunk 3]
```diff
@@ -708,6 +709,7 @@ focus_carousing = {
 		can_rule_peacefully_trigger = yes
 		evil_public_religion_trigger = no
 		is_uncursed_titanforged_trigger = no
+		is_plant_race = no
```
Comment (zumbak04) [inline common/objectives/00_focuses.txt]: is_plant_race trigger doesn't exists.
```suggestion
		plant_race = no
```

File: common/objectives/00_focuses.txt (hunk 1 above)
Comment (sejtalk) [reply common/objectives/00_focuses.txt]: fixed

File: common/objectives/00_focuses.txt (hunk 2 above)
Comment (sejtalk) [reply common/objectives/00_focuses.txt]: fixed

File: common/objectives/00_focuses.txt (hunk 3 above)
Comment (sejtalk) [reply common/objectives/00_focuses.txt]: fixed

**Pull request merged**